### PR TITLE
Update isyncr to 5.14.0

### PR DESCRIPTION
**Pull request by brianbrownton**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by brianmorton** [comment]
Actually pls hold on merging this - there are different versions for different macOS versions.

**Comment by brianmorton** [comment]
Should be good to merge now, if you all are happy with it.

**Comment by vitorgalvao** [comment]
Why drop the appcast?

**Comment by brianmorton** [comment]
@vitorgalvao 

It is unusual for an appcast to point to a JS file ( http://www.jrtstudio.com/files/SlashiSyncr25.js ), so I had a look.

The contents of the file seem to facilitate downloading from an HTML page, however all the version numbers were wrong/outdated - the mac version was for `5.0.8` - which was the lowest common denominator in the mac versions (lion and older). So clearly it was no longer being updated for recent versions, and it really never should have served as an "appcast".

**Comment by brianmorton** [comment]
To expand further:

I looked at a candidate for a new appcast. The equivalent up to date file of the old appcast is http://www.jrtstudio.com/files/SlashiSyncr32.js , however that doesn't seem to correlate to the version numbers (specifically the `32` at the end) so it would be hard to tell if/when it became outdated, like the old appcast.

I also looked at the changelog ( http://www.jrtstudio.com/iSyncr/Changelog ), however even that is outdated compared to the version offered for download (`5.12` on the changelog vs `5.14` for download) - not a great start. After that, I have had issues with using a webpage for appcasts before due to the risk of dynamic content, so since it was outdated and I didn't want to run that risk I just felt it was better to drop the appcast.

I think no information is better than bad information.

File: Casks/isyncr.rb [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
+    sha256 '68e65554b12083c35b0180c6535cefcc7897c0aa4902848dcf706459ecfab3ad'
+  end
 
   url "http://www.jrtstudio.com/files/iSyncr%20Desktop%20#{version}.pkg"
-  appcast 'http://www.jrtstudio.com/files/SlashiSyncr25.js',
-          checkpoint: '65385df99037be7b37ea118f503a51036b2af6c11a3c6230ff5b4f3576dee4ab'
   name 'iSyncr Desktop'
   homepage 'http://www.jrtstudio.com/iSyncr-iTunes-for-Android'
 
-  depends_on macos: '>= :mountain_lion'
```
Comment (commitay) [inline Casks/isyncr.rb]: Lets drop the lion version and keep the depends_on mountain_lion

Comment (brianmorton) [reply Casks/isyncr.rb]: With respect, I believe 10.7 is Lion and I was just trying to follow what is listed on the site's download page:

image

Based on the bit about
> Mac 10.7 & older

I dropped the `depends_on` stanza.

Comment (commitay) [reply Casks/isyncr.rb]: We don't need to have a 10.7 version just because the homepage has one.

Comment (brianmorton) [reply Casks/isyncr.rb]: Right... well now I'm not really clear on what exactly you want changed. So I will leave this PR as it is, and you can either close it or modify it to suit your goal.

**Comment by commitay** [comment]
Thanks @vitorgalvao and apologies for being unclear @brianmorton

**Pull request merged**